src/core/edit/edit.cell.view.js

```diff
@@ -109,7 +117,16 @@ export class EditCellView {
 					}
 
 					cell = cell || this.editor.cell || model.navigation().cell;
-					if (cell && model.edit().cancel.execute(this.contextFactory(cell, this.value, this.label)) !== false) {
+					let label = this.label;
```

Comment:
don't understand this code, why we need to anything on cancel? it should just close editor